test/test_image.rb

```diff
@@ -14,7 +14,7 @@ def setup
 
   def test_image
     test_scarpe_code_no_display(<<~SCARPE_APP, <<~'TEST_CODE')
-      Shoes.app do
+      Scarpe.app do
```

Comment:
For consistency we're trying to keep Shoes.app as what we're using. was that breaking here?

Reply:
i will change that